app/decorators/application_decorator.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationDecorator < Draper::Decorator
```

Comment:
Porque criou uma classe base? Vendo abaixo o `cidadao_decorator.rb` isso só faz sentido se o CidadaoDecorator for herdado de ApplicationDecorator. Classes bases são bem legais pra vc restringir a lógica dentro de vários contextos (namespaces por exemplo). Nesse caso simples, recomendo a remoção.

Reply:
Acho que essa classe é criada quando instalo o draper, vou remover então.